pinc/wordcheck_engine.inc

```diff
@@ -614,7 +608,7 @@ function count_wordcheck_suggestion_events($projectid, $start_time=0)
             WHERE projectid='$projectid'
```

Comment:
We need to escape `$projectid` and ensure `$start_time` is a number via sprintf. Also we should raise an exception on error (the FALSE check is wrong in the original code).

Reply:
Done.